packages/components-react/card-react/src/index.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2020-2025 Frameless B.V.
+ * Copyright (c) 2021-2025 Gemeente Utrecht
+ */
+
+import { Link as UtrechtLink } from '@utrecht/link-react';
+import clsx from 'clsx';
+import { ForwardedRef, forwardRef, PropsWithChildren, useId, useRef } from 'react';
+import type { ElementType, HTMLAttributes, KeyboardEvent, ReactNode } from 'react';
+export interface HTMLHeadingProps extends PropsWithChildren<HTMLAttributes<HTMLHeadingElement>> {
+  level: number;
+}
+
+export const HTMLHeading = ({ level, ...restProps }: HTMLHeadingProps) => {
+  const HeadingX =
+    level === 2 ? 'h2' : level === 3 ? 'h3' : level === 4 ? 'h4' : level === 5 ? 'h5' : level === 6 ? 'h6' : 'h1';
+
+  return <HeadingX {...restProps} />;
+};
+export type HTMLHeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
+export interface CardProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Optional image solt to display at the top of the card
+   */
+  image?: ReactNode;
+  /**
+   * Heading content - can be a string or a ReactNode
+   */
+  heading: ReactNode;
+  /**
+   * Heading level (1-6, default to 2)
+   */
+  headingLevel: HTMLHeadingLevel;
+  /**
+   *  URL for the heading link
+   */
+  href?: string;
+  /**
+   * Additional CSS classes for the Card Link
+   * */
+  linkClassName?: string;
```

Comment:
Als mensen class names op de link wil, dan moeten mensen maar een `Link={(...restProps) => <UtrechtLink className="custom" {...restProps}>` wrapper maken. Is dat niet voldoende? Want waarom zouden we specifiek `className` aanbieden, en niet alle andere properties?

Reply:
You're right! Your approach will be sufficient!